tests/integration_tests/clouds.py

```diff
@@ -173,6 +173,9 @@ def launch(self, user_data=None, launch_kwargs=None, wait=True,
                 'cloud-init version: %s',
                 instance.execute("cloud-init --version")
             )
+            serial = instance.execute("grep serial /etc/cloud/build.info")
+            if serial:
```

Comment:
Nit: I think `serial.ok` would be slightly better than relying on if something exists in stdout, but this should be ok too.